exercises/knapsack/description.md

```diff
@@ -0,0 +1,35 @@
+In this exercise, lets try to solve a classic problem.
+
+Bob is a thief. After months of careful planning, he finally manages to
+crack the security systems of a high-class apartment.
+
+In front of him are many items, each with a value (v) and weight (w). Bob,
+of course, wants to maximize the total value he can get; he would gladly
+take all of the items if he could. However, to his horror, he realizes that
+the knapsack he carries with him can only hold so much weight (W).
+
+Given a knapsack with a specific carrying capacity (W), help Bob determine
+the maximum value he can get from the items in the house. Note that Bob can
+only take one of each item.
```

Comment:
~~only~~ take only one of each item.

Because if Bob can only take one of each item it does not allow the option of taking only one of each item.

Reply:
Yes good catch! I have updated this.